pandas/io/json/normalize.py

```diff
@@ -111,6 +111,8 @@ def json_normalize(data, record_path=None, meta=None,
     record_path : string or list of strings, default None
         Path in each object to list of records. If not passed, data will be
         assumed to be an array of records
+        For arrays of inconsistantly formatted json records, first record
```

Comment:
Inconsistently

Reply:
Generally not sure of the wording here. Inconsistent isn't really the right term as I think it implies that the JSON is malformed or invalid, which isn't the case.

What distinction are you trying to make with the first record?

Reply:
Updated to
_**For an array of objects with missing key-value pairs in each record,
the first record needs to include all key-value pairs**_

The point being that if all the keys aren't seen in the first record, its not straightforward to then fill in previous records that have been loaded with NaNs. Easier for the first record to provide a 'template' of all keys, so that as each subsequent record is parsed, the dataframe can be easily loaded with NaNs if the key does not exist.